specs/LaunchingRegisteredProtocols.md

```diff
@@ -0,0 +1,316 @@
+  /// or replace the default dialog with a custom dialog.
+  ///
+  /// If a deferral is not taken on the event args, the external protocol launch is 
+  /// blocked until the event handler returns.  If a deferral is taken, the
+  /// external protocol launch is blocked until the deferral is completed.
+  /// The host also has the option to cancel the protocol launch.
+  ///
+  /// The `NavigationStarting` and `NavigationCompleted` events will be raised,
+  /// regardless of whether the `Cancel` property is set to `TRUE` or
+  /// `FALSE`. The `SourceChanged`, `ContentLoading`, and `HistoryChanged` events 
+  /// will not be raised regardless of this property. 
+  /// The `LaunchingExternalProtocol` event will be raised after the 
+  /// `NavigationStarting` event and before the `NavigationCompleted` event.
+  /// The default settings will also be updated upon navigation to an external
+  /// protocol. 
+  ///
+  /// If the request is made from a trustworthy origin
+  /// (https://w3c.github.io/webappsec-secure-contexts#potentially-trustworthy-origin) 
+  /// a checkmark box will be displayed on the default browser UI that gives the user 
+  /// the option to always allow the external protocol to launch from this origin.
+  /// If the user checks this box, upon the next request from that origin to the 
+  /// protocol, the event will still be raised. 
+  ///
+  /// If the request is initiated by a cross-origin iframe without a user gesture, 
+  /// the request will be blocked and the `LaunchingExternalProtocol` event will not 
+  /// be raised. 
+  /// If the request is initiated by a browser popup window or in another case in 
+  /// which the embedded browser is null, the `LaunchingExternalProtocol` event will 
+  /// not be raised and the request will continue as normal and display the default browser
+  /// UI dialog to the user.
+  /// \snippet SettingsComponent.cpp LaunchingExternalProtocol
+  HRESULT add_LaunchingExternalProtocol(
+      [in] ICoreWebView2LaunchingExternalProtocolEventHandler* eventHandler,
+      [out] EventRegistrationToken* token);
+
+  /// Remove an event handler previously added with
+  /// `add_LaunchingExternalProtocol`.
+  HRESULT remove_LaunchingExternalProtocol(
+      [in] EventRegistrationToken token);
+  }
+
+/// Receives the `LaunchingExternalProtocol` event.
+[uuid(e5fea648-79c9-47aa-8314-f471fe627649), object, pointer_default(unique)] 
+interface ICoreWebView2LaunchingExternalProtocolEventHandler: IUnknown {
+  /// Provides the event args for the corresponding event.
+  HRESULT Invoke(
+      [in] ICoreWebView2* sender,
+      [in] ICoreWebView2LaunchingExternalProtocolEventArgs* args);
+}
+
+/// Event args for `LaunchingExternalProtocol` event.
+[uuid(fc43b557-9713-4a67-af8d-a76ef3a206e8), object, pointer_default(unique)] 
+interface ICoreWebView2LaunchingExternalProtocolEventArgs: IUnknown {
+  /// The URI of the requested external protocol.
+
+  [propget] HRESULT Uri([out, retval] LPWSTR* uri);

```

Comment:
The parameter names of property methods should be `value`. True throughout the doc
```suggestion
  [propget] HRESULT Uri([out, retval] LPWSTR* value);
```